packages/shared/src/components/post/MarkdownPostContent.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactElement } from 'react';
+import { Post, PostType } from '../../graphql/posts';
+import Markdown from '../Markdown';
+import { LazyImage } from '../LazyImage';
+import { cloudinary } from '../../lib/image';
+import { Image } from '../image/Image';
+
+interface WelcomePostContentProps {
+  post: Post;
+}
+
+function WelcomePostContent({ post }: WelcomePostContentProps): ReactElement {
+  return (
+    <>
+      <h1 className="my-6 font-bold whitespace-pre-line typo-title2">
+        {post.title}
+      </h1>
+      {post.type === PostType.Freeform && (
+        <Image
+          src={post.image}
+          alt="Post cover image"
+          className="object-cover mb-10 w-full h-auto rounded-xl max-h-[18rem]"
+          fallbackSrc={cloudinary.post.imageCoverPlaceholder}
+        />
+      )}
```

Comment:
This was just a renamed file with the only change being highlighted above, plus the render condition on the image below.